File: PocketBeagle/TechLab/internalLED.py
#!/usr/bin/python3
#//////////////////////////////////////
#	internalLED.py
#	Blinks the USR3 internal LED.
#//////////////////////////////////////
import Adafruit_BBIO.GPIO as GPIO
import time

out = "USR3"
 
GPIO.setup(out, GPIO.OUT)
 
while True:
    GPIO.output(out, GPIO.HIGH)
    time.sleep(0.25)
    GPIO.output(out, GPIO.LOW)
    time.sleep(0.25)

File: PocketBeagle/TechLab/internalLED.js
#!/usr/bin/env node
//////////////////////////////////////
// 	internaLED.js
// 	Blinks the internal USR3 LED.
//////////////////////////////////////
var fs = require('fs');
var LED = '/sys/class/leds/beaglebone:green:usr3/brightness';
var state = 0;     // Initial state

setInterval(flash, 250); // Change state every 250 ms

function flash() {
  fs.writeFileSync(LED, state);
  if(state == 1) {
    state = 0;
  } else {
    state = 1;
  }
}
